src/messages/MessageElement.cpp

```diff
@@ -395,6 +422,138 @@ void TextElement::addToContainer(MessageLayoutContainer &container,
     }
 }
 
+SingleLineTextElement::SingleLineTextElement(const QString &text,
+                                             MessageElementFlags flags,
+                                             const MessageColor &color,
+                                             FontStyle style)
+    : MessageElement(flags)
+    , color_(color)
+    , style_(style)
+{
+    for (const auto &word : text.split(' '))
+    {
+        this->words_.push_back({word, -1});
+        // fourtf: add logic to store multiple spaces after message
```

Comment:
Updated in e0cabce

I had left this comment in because I wasn't sure what fourtf was trying to do, but because a SingleLineTextElement doesn't have anything after it, I decided to just remove this comment.